### PR TITLE
Fix compatibility with poetry 1.5+

### PR DESCRIPTION
**Pull request by frenzymadness**
Poetry 1.5 no longer provides category info in poetry.lock.

Fixes: https://github.com/thoth-station/micropipenv/issues/280

Cc @abompard

### CODE REVIEW

File: tests/data/parse/poetry_1_5/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "aaa"
+version = "0.1.0"
+description = ""
+authors = ["Fridolin Pokorny <fridolin@redhat.com>"]
```
Comment (goern) [inline tests/data/parse/poetry_1_5/pyproject.toml:5]: @fridex wanna give this a new email?

**Review by goern — APPROVED** [review]

File: tests/data/parse/poetry_1_5/pyproject.toml (hunk 1 above)
Comment (fridex) [reply tests/data/parse/poetry_1_5/pyproject.toml:5]: It's fine, at least spam bots will not reach out to me :)

**Comment by isaac-florence** [comment]
This fix will be helpful for us being able to upgrade beyond 1.4.2 - is there a date for the next release of poetry? Thanks!

**Comment by frenzymadness** [comment]
Sorry for the delay. I had a plan to fix more bugs before the next release but I have no time so I'm going to release a new version now.

**Pull request merged**